interface/src/app/about/about.tpl.html

```diff
@@ -1,8 +1,6 @@
-<div>
-  <h1 class="page-header">
-    About ADAGE
-  </h1>
-  <p>
-    This is where some explanatory text about ADAGE should go.
-  </p>
+<div class="page-header">
+  <h1>About ADAGE</h1>
 </div>
+<p>
+  This is where some explanatory text about ADAGE should go.
+</p>
```

Comment:
How about:

`The ADAGE webserver allows users without programming expertise to analyze signatures extracted from large compendia of gene expression data via unsupervised machine learning methods. We primarily designed the server to support models constructed by [ADAGE](https://doi.org/10.1128/mSystems.00025-15) and [eADAGE](https://doi.org/10.1101/078659); however, the server was designed to be general and can be used for other types of machine learning models as well.